### PR TITLE
Token instance page: general video improvements

### PR DESCRIPTION
**Pull request by vbaranov**
Finalizes https://github.com/blockscout/blockscout/pull/5296

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [x] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [x] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [x] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [x] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

**Comment by cen1** [comment]
@vbaranov just to double check, the reason I split get_media_src is to avoid loading full videos in token inventory page. If you have 20 tokens in the listing and all fetch the full video the load is incredibly slow in my experience. I think this was lost in the current merge if I read correctly.

TLDR, what I believe is the optimal solution:
1. Keep the old logic for token inventory page (usually it loads low quality gifs)
2. Fetch animation_url on detail page for maximum experience (if it exists, cascade down must remain ofc).

Function split is probably not necessary by passing a parameter into it, I just didn't know how to do that on my own.

**Comment by vbaranov** [comment]
> the reason I split get_media_src is to avoid loading full videos in token inventory page

@cen1 good point. I will address it.

**Comment by vbaranov** [comment]
@cen1 here https://github.com/blockscout/blockscout/pull/5305

**Comment by cen1** [comment]
Looks good, thank you @vbaranov 

Will test and come back if anything is amiss
